docs/rules/prefer-ember-test-helpers.md

```diff
@@ -0,0 +1,66 @@
+# prefer-ember-test-helpers
+
+This rule ensures the correct Ember test helper is imported when using methods with `await` that have a native window counterpart.
```

Comment:
I'm not sure we should limit this lint rule to only checking for the `await` usages of these functions? Why do you think we should limit it like that? I'm leaning towards expanding it to any usage of the functions.

Reply:
Hmm, I had thought the issue was specifically occurring when trying to use in conjunction with `await` in that Ember test helpers will handle these asynchronous methods for you and not if they're not imported. I didn't see anyone in our current codebase using it otherwise, so made an assumption - do you happen to have some context on how else it's being used? :)

Reply:
Ideally the functions should be used with `await`, but I don't think we should add this limitation to the rule. I'd rather the rule applies to any of the relevant function calls regardless of how they are used. There could be other valid usages of them.

Reply:
That sounds good! The linter doesn't look for `await` specifically, should I add some additional test cases that don't use await?

Reply:
We can use `emberUtils.isTestFile` to ensure this rule only applies to test files. Since any code in test files should be using `blur` and similar functions from `@ember/test-helpers` right? That would replace the `await` check.

Then you will actually want to update the tests to ensure the rule only applies to test files, instead of looking for await.